leasing/report/lease/lease_statistic_report.py

```diff
@@ -0,0 +1,398 @@
+            return True
+    return False
+
+
+def get_buy_right(obj):
+    for decisions in obj.decisions.all():
+        if(decisions.conditions.filter(type_id=24)):  # TODO: Can we get rid of static type_id
+            return True
+    return False
+
+
+def get_lease_area_identifier(obj):
+    lease_area_identifiers = []
+
+    for lease_area in obj.lease_areas.all():
+        if lease_area.archived_at:
+            continue
+
+        lease_area_identifiers.extend([lease_area.identifier])
+
+    return ' / '.join(lease_area_identifiers)
+
+
+def get_total_rent_amount_for_year(obj):
+    year = datetime.date.today().year
+    return obj.calculate_rent_amount_for_year(year).get_total_amount()
+
+
+def get_total_area(obj):
+    return obj.lease_areas.aggregate(Sum('area'))['area__sum']
+
+
+def get_re_lease(obj):
+    year = datetime.date.today().year
+    year_start = datetime.date(year, 1, 1)
+    year_end = datetime.date(year, 12, 31)
+    for rent in obj.get_active_rents_on_period(year_start, year_end):
+        if(rent.rent_adjustments.filter(subvention_type=SubventionType.RE_LEASE)):
+            return True
+    return False
+
+
+def get_build_permission_living(obj):
+    qs = obj.basis_of_rents.filter(intended_use__id__in=LIVING).values('area_unit').annotate(area=Sum('area'))
+    return ' / '.join(['{} {}'.format(i.get('area'), i.get('area_unit')) for i in qs])
+
+
+def get_build_permission_business(obj):
+    qs = obj.basis_of_rents.exclude(intended_use__id__in=LIVING).values('area_unit').annotate(area=Sum('area'))
+    return ' / '.join(['{} {}'.format(i.get('area'), i.get('area_unit')) for i in qs])
+
+
+def get_build_permission_total(obj):
+    qs = obj.basis_of_rents.all().values('area_unit').annotate(area=Sum('area'))
+    return ' / '.join(['{} {}'.format(i.get('area'), i.get('area_unit')) for i in qs])
+
+
+def get_rent_amount_living(obj):
+    year = datetime.date.today().year
+    sum = 0
+    for a in obj.calculate_rent_amount_for_year(year).amounts:
+        for tabiu, value in a.get_total_amounts_by_intended_uses().items():
+            if(tabiu.id in LIVING):
+                sum += value
+    return sum
+
+
+def get_rent_amount_business(obj):
+    year = datetime.date.today().year
+    sum = 0
+    for a in obj.calculate_rent_amount_for_year(year).amounts:
+        for tabiu, value in a.get_total_amounts_by_intended_uses().items():
+            if(tabiu.id not in LIVING):
+                sum += value
+    return sum
+
+
+def get_rent_amount_for_year(obj):
+    year = datetime.date.today().year
+    return obj.calculate_rent_amount_for_year(year).get_total_amount()
+
+
+def get_floor_m2_living(obj):
+    qs = obj.basis_of_rents.filter(intended_use__id__in=LIVING).values(
+        'area_unit').annotate(amount_per_area=Avg('amount_per_area'))
+    return ' / '.join(['{} {}'.format(i.get('amount_per_area'), i.get('area_unit')) for i in qs])
```

Comment:
So at the moment a value for this field might look something like `100 m2 / 200 kem2 / 50 m2`, right? I think it's good to merge for testing in the QA environment, but let's make sure that this is what is needed when Katja has had a chance to see it in action (and that they understand the contents, i.e. average area grouped by area unit).